src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -32,6 +32,13 @@ typedef struct {
 #define CHEST_ANIM_SHORT 0
 #define CHEST_ANIM_LONG 1
 
+typedef enum {
+    /* 0x00 */ HANDLE_ON_LEFT_ADULT,
+    /* 0x01 */ HANDLE_ON_LEFT_CHILD,
+    /* 0x02 */ HANDLE_ON_RIGHT_ADULT,
+    /* 0x03 */ HANDLE_ON_RIGHT_CHILD
+} DoorAnimStyle;
```

Comment:
im unsure if animation style is the term we wanna go with here. can you go into detail on what that means?
we'd also want this enum to have a prefix reflecting what it is, whenever we agree on a term for it

Reply:
specifically for doors with handles, link has 4 different animations he can do depending on which side the door handle is on and if he's adult or child. This is used to index which animation to play (it's common across EnDoor and DoorKiller)

I didn't like the word "style" either but I wasn't able to come up with anything better. Maybe doorHandleAnimation?

Reply:
knobDoorAnim, knobbedDoorAnim, something along those lines could work, or handle if you like that better. def think dropping the word style and calling it an animation on its own is best either way

Reply:
Was trying to avoid the word "Knobbed" cause it's a vulgar term in British slang lol

Would you call the enum entries then KNOBBED_DOOR_ANIM_LEFT_KNOB_ADULT etc? Seems a bit long

Reply:
KNOB_ANIM_ADULT_R, L for left
prob can assume that knob will imply its for a door. just throwing out random ideas tho maybe others have suggestions